python-modules-list.sh

```diff
@@ -76,6 +76,27 @@ env:
     dryable==1.0.5
     responses==0.10.6
     pandas==1.1.5
+  # Keep the PIPxy version in sync with the Conda env we install in Python-modules!
+  # Everything but the first two lines copied from PIP39_REQUIREMENTS, but with versions
+  # adjusted such that wheels are available and for compatibility with tensorflow.
+  PIP39_REQUIREMENTS_osx_arm64: |
+    tensorflow-macos==2.12.0
+    tensorflow-metal==0.8.0
+    PyYAML==5.4.1
+    psutil==5.9.5
+    uproot==4.1.0
+    numpy==1.23.5
+    scipy==1.10.1
+    Cython==0.29.21
+    seaborn==0.11.0
+    scikit-learn==1.2.2
+    sklearn-evaluation==0.12.0
+    Keras==2.12.0
+    xgboost==1.7.5
+    dryable==1.0.5
+    responses==0.10.6
+    pandas==1.5.3
+    setuptools==65.5.1
```

Comment:
Hi @TimoWilken given that setuptools is installed as common dependency in #4972 maybe here is redundant?

Reply:
Good point @adriansev. I'll rebase this PR on top of yours once yours is merged and fix stuff like this!